app/controllers/StoriesVisibleController.scala

```diff
@@ -12,12 +13,16 @@ case class StoriesVisibleRequest(
   stories: Seq[Story]
 )
 
-class StoriesVisibleController(val containerService: ContainerService, val deps: BaseFaciaControllerComponents) extends BaseFaciaController(deps) {
+class StoriesVisibleController(val containerService: ContainerService, val deps: BaseFaciaControllerComponents) extends BaseFaciaController(deps) with Logging {
   def storiesVisible(containerType: String) = AccessAPIAuthAction(parse.json[StoriesVisibleRequest]) { implicit request =>
     val storiesVisible = containerService.getStoriesVisible(containerType, request.body.stories)
 
+    logger.info(s"got stories-visible=$storiesVisible for containerType=$containerType")
+
     storiesVisible.map { storiesVisibleResponse => Ok(Json.toJson(storiesVisibleResponse)) } getOrElse {
-      NotFound(s"$containerType is not a valid container id")
+      val errorMSG = s"'$containerType' is not a valid container id"
```

Comment:
Type or id?  I think these are different. Can you check this error message is correct please?

Reply:
will change :) that log was like that before, but it make sense